lib/codecs/UriDecoder.js

```diff
@@ -11,16 +12,18 @@ const SCHEME_ID_TO_DECODER = new Map([
   [2, HttpUrlDecoder],
   [3, UuidUrnDecoder],
   [1024, Base58DidUrlDecoder],
-  [1025, Base58DidUrlDecoder]
+  [1025, Base58DidUrlDecoder],
+  // FIXME: encdeds as 3 bytes, may want to use a lower id
+  [1026, AppUrlDecoder]
```

Comment:
Seems like we could use `4` here without issue, right?

Reply:
I'm not sure what the policy is here.  This is another registry of ids.  This app url codec seems general enough to use a limited low id. Perhaps more specialized codecs don't?  I'm guessing that's why the base58did ones are higher?